8th-snippets.code-snippets

```diff
@@ -0,0 +1,51 @@
+{
+	// Place your forth-language workspace snippets here. Each snippet is defined under a snippet name and has a scope, prefix, body and 
+	// description. Add comma separated ids of the languages where the snippet is applicable in the scope field. If scope 
+	// is left empty or omitted, the snippet gets applied to all languages. The prefix is what is 
+	// used to trigger the snippet and the body will be expanded and inserted. Possible variables are: 
+	// $1, $2 for tab stops, $0 for the final cursor position, and ${1:label}, ${2:another} for placeholders. 
+	// Placeholders with the same ids are connected.
+	// Example:
+	// "Print to console": {
+	// 	"scope": "javascript,typescript",
+	// 	"prefix": "log",
+	// 	"body": [
+	// 		"console.log('$1');",
+	// 		"$2"
+	// 	],
+	// 	"description": "Log output to console"
+	// }
+	"if then clause": {
+		"scope": "8th",
+		"prefix": "ift",
+		"body": [
+			"${1:condition} if",
+			"\t${2:code}",
+			"then",
+			"$0"
+		]
+	},
+	"if then else clause": {
+		"scope": "8th",
+		"prefix": "ife",
+		"body": [
+			"${1:condition} if",
+			"\t${2:if-code}",
+			"else",
+			"\t${3:else-code}",
+			"then",
+			"$0"
+		]
+	},
+	"word definitiom": {
+		"scope": "8th",
+		"prefix": "def",
+		"body": [
+			"\\ ${1:description}",
```

Comment:
Instead of `\ {description}`, it'd probably be better to add the full word documentation from 8th:
```
-- w: ${2:word}
-- s: ${3:before} -- ${4:after}
-- d: ${1:description}
```
This way, if Ron ever adds support for his documentation-generating code to be used elsewhere, it'll start picking these up.